packages/0x.js/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+v0.28.0 - _TBD_
+------------------------
+* Add `etherTokenAddress` arg to `depositAsync` and `withdrawAsync` methods on `EtherToken` wrapper. (#267)
```

Comment:
on `zeroEx.etherToken`. The fact that it's a wrapper is an internat knowledge, but changelog is for external users.

Reply:
Also - document two other changes. Removal of public method unsubscribeAll and removal of config parameter in top-level ZeroExConfig,